{"name": "test_add2[【类型】输入英文，给出提示信息1]", "status": "passed", "steps": [{"name": "步骤一：nu和0.2开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\study0818\\test\\test_calculate.py\", line 52, in test_add2\n    result=self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1660804986829, "stop": 1660804986829}], "attachments": [{"name": "log", "source": "13ac2d1d-5ffe-4f3d-b6f4-29a2870cc7d3-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "2ba03a0c-582e-4deb-9087-73f8f8a691c4-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "05212b44-f34e-4766-8828-0530586b0fd4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "'nu'"}, {"name": "y", "value": "0.2"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1660804986829, "stop": 1660804986829, "uuid": "cba791a9-60f6-4105-9216-1060113ac9c0", "historyId": "daa55f79c6b0e8de4e9c0aca5cec7405", "testCaseId": "c1fd4152c4894ab47d6f7448dc1c5bcf", "fullName": "study0818.test.test_calculate.TestAdd#test_add2", "labels": [{"name": "story", "value": "加法报错用例"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "study0818.test"}, {"name": "suite", "value": "test_calculate"}, {"name": "subSuite", "value": "TestAdd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "13372-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "study0818.test.test_calculate"}]}